modules/ddl-and-loading/pages/modifying-a-graph-schema.adoc

```diff
@@ -28,6 +28,58 @@ Load or query operations which begin before the schema change will be completed
 Schema changes that are performed during times with high query or data loading volume may have a chance of failure.
 It is considered good practice to perform schema changes during times of low graph activity.
 
+== Dynamic Schema Change - Abilities and Limitations
+
+Dynamic Schema Change (DSC) acts as a traffic manager for schema change jobs.  First, if there are any active loading jobs in progress when a schema change is requested, and the schema change affects a graph element included in the loading job, DSC should pause the schema change until the loading job completes.
+
+Once the schema change begins, DSC enables a modest amount of database activity (queries and loading) during a schema change, but ONLY if those operations do not involve graph elements affected by the schema change. If the activity level is too high, there is a risk of either the schema change job not succeeding, the concurrent database activity not succeeding, or both.
+
+[WARNING]
+====
+For maximum safety, is it best to halt all graph activity before and during a schema change. If a schema change job fails, it can leave the graph in an inconsistent state.
+====
+
+=== Impact on Queries and Loading Jobs
+
+[IMPORTANT]
+====
+A schema change will invalidate any loading jobs or query which relate to an altered part of the schema. 
+====
+ 
+* A loading job becomes invalid if it references a vertex or edge that has been DROPPED or ALTERED. 
+* A query becomes invalid if it refers to a DROPPED vertex, edge, or attribute.
+
+Invalid loading jobs are *DROPPED*, and invalid queries are *uninstalled*. After the schema update, users must create and install new loading and query jobs based on the updated schema.
+
+*What to Do:*
+
+* After schema changes, reinstall and redefine queries and loading jobs based on the new schema.
+* Ensure that the updated queries and jobs align with the modified schema and are properly tested to ensure they work as expected.
```

Comment:
```suggestion
```
This part is (1) repeating what you said earlier and (2) applies to all schema changes, not just DSC.
That is, a query or loading job does not have to be running in order for it to be invalidated.
So, I would remove this.  But you can see whether you think your wording was better here or above and use the one with better wording.